pandas/io/formats/format.py

```diff
@@ -1621,18 +1621,23 @@ def __init__(
 
     def _format_strings(self) -> list[str]:
         """we by definition have DO NOT have a TZ"""
-        values = self.values
 
+        values = self.values
         if not isinstance(values, DatetimeIndex):
             values = DatetimeIndex(values)
```

Comment:
OK, so there is a deeper bug here in that DatetimeIndex(values) isn't raising when passed a 2D array.